test/data/test_builtin_datasets.py

```diff
@@ -82,13 +89,14 @@ def test_penntreebank(self):
         tokens_ids = [vocab[token] for token in 'the player characters rest'.split()]
         self.assertEqual(tokens_ids, [2, 2550, 3344, 1125])
 
-        # Delete the dataset after we're done to save disk space on CI
-        datafile = os.path.join(self.project_root, ".data", 'ptb.train.txt')
-        conditional_remove(datafile)
-        datafile = os.path.join(self.project_root, ".data", 'ptb.test.txt')
-        conditional_remove(datafile)
-        datafile = os.path.join(self.project_root, ".data", 'ptb.valid.txt')
-        conditional_remove(datafile)
+        if os.environ.get("TRAVIS") == "true":
```

Comment:
Since we got rid of Travis, do we still need this?

Reply:
removed!